swift/Sources/SignalClient/Error.swift

```diff
@@ -92,3 +96,19 @@ internal func checkError(_ error: SignalFfiErrorRef?) throws {
         throw SignalError.unknown(errType, errStr)
     }
 }
+
+internal func assertNoError(_ error: SignalFfiErrorRef?) {
+    assertNoError { try checkError(error) }
```

Comment:
Feel free to ignore, but I personally would rename this to something like `failOnError`.